File: python/01_estrutura_sequencial/ex08.py
"""
Exercício 08

Faça um Programa que pergunte quanto você ganha por hora e o número de horas trabalhadas
no mês. Calcule e mostre o total do seu salário no referido mês.
"""

salario_hora = float(input("Salário hora R$ "))
horas_trabalhadas = float(input("Horas trabalhadas: "))
salario = round(salario_hora * horas_trabalhadas, 2)
print(f"Salário mensal R$ {salario:.2f}")

File: nodejs/01-estrutura-sequencial/ex08.js
/*
 * Exercício 08
 *
 * Faça um Programa que pergunte quanto você ganha por hora e o número de horas trabalha-
 * das no mês. Calcule e mostre o total do seu salário no referido mês.
 */

import * as readlinePromises from 'node:readline/promises'
import {stdin as input, stdout as output} from 'node:process'

const rlp = readlinePromises.createInterface({input, output})

const salarioHora = parseFloat(await rlp.question("Salário hora R$ "))
const horasTrabalhadas = parseFloat(await rlp.question("Horas trabalhadas: "))

rlp.close()

const salario = salarioHora * horasTrabalhadas
console.log(`Salário mensal R$ ${salario.toFixed(2)}`)
